CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## __AWS SDK for Java v2__
```

Comment:
The changelog is actually auto-generated. Adding a new entry can be done by running the `scripts/new-change `script and following the instructions. Commit the new file created by the script in .changes/next-release with your changes.